src/human-readable/runtime/cache.ts

```diff
@@ -1,5 +1,8 @@
 import type { AbiParameter } from '../../abi'
 
+export function getParameterCacheKey(param: string, type?: string) {
+  return `${type ? `${type}:` : ''}${param}`
+}
```

Comment:
idk why i made this a ternary. this is much nicer imo.

```suggestion
export function getParameterCacheKey(param: string, type?: string) {
  if (type) return `${type}:${param}`
  return param
}
```